saskatoon/harvest/forms.py

```diff
@@ -593,6 +609,20 @@ def clean_end_date(self):
             )
         return end_dt
 
+    def save(self):
+        """"
+        Convert list of autocomplete equipment points into all equipment owned by said equipment points.
+        i.e. Reserving an equipment point will reserve all its equipment for the duration of the harvest.
+        """
+
+        instance = super(HarvestForm, self).save(commit=False)
+        equipment_points = self.cleaned_data["equipment_point"]
+        equipment_owned_by_equipment_points = Equipment.objects.filter(owner__in=equipment_points)
```

Comment:
the wording isn't clear. We don't need to know about the autocomplete here and `self.cleaned_data["equipment_point"]` is a one equipment point (i.e. a single `Organization` object)